src/server/game/Scripting/ScriptMgr.h

```diff
@@ -908,7 +915,7 @@ class PlayerScript : public ScriptObject
     virtual void OnBeforeUpdateAttackPowerAndDamage(Player* /*player*/, float& /*level*/, float& /*val2*/, bool /*ranged*/) { }
     virtual void OnAfterUpdateAttackPowerAndDamage(Player* /*player*/, float& /*level*/, float& /*base_attPower*/, float& /*attPowerMod*/, float& /*attPowerMultiplier*/, bool /*ranged*/) { }
 
-    virtual void OnBeforeInitTalentForLevel(Player* /*player*/, uint8& /*level*/, uint32& /*talentPointsForLevel*/) { }
+    virtual void OnBeforeInitTalentForLevel(Player* /*player*/, uint32& /*talentPointsForLevel*/) { }
```

Comment:
Also, add this change to the changelog... Something like, for hook OnBefore.... it doesn't require the parameter level anymore.